charms/slurmd/src/constants.py

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Omnivector, LLC
+
+"""Slurmd Charm Constants."""
+
+CHARM_MAINTAINED_NHC_CONFIG = """
```

Comment:
[nit]: could we shorten this constant to just `NHC_CONFIG`? Less typing and less possible matches with other constants prefixed with `CHARM_MAINTAINED_` :sweat_smile: 

```suggestion
NHC_CONFIG = """
```